{
	"name": "CVE-2021-3588",
	"threat_severity": "Low",
	"public_date": "2021-01-04T00:00:00Z",
	"bugzilla": {
		"description": "out-of-bounds read in cli_feat_read_cb() in src/gatt-database.c",
		"id": "1970592",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1970592"
	},
	"cvss3": {
		"cvss3_base_score": "3.3",
		"cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N",
		"status": "draft"
	},
	"cwe": "CWE-125",
	"details": [
		"The cli_feat_read_cb() function in src/gatt-database.c does not perform bounds checks on the 'offset' variable before using it as an index into an array for reading."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2021-3588\nhttps://nvd.nist.gov/vuln/detail/CVE-2021-3588"
	],
	"package_state": [
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:6",
			"fix_state": "Out of support scope",
			"package_name": "bluez",
			"product_name": "Red Hat Enterprise Linux 6"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:7",
			"fix_state": "Out of support scope",
			"package_name": "bluez",
			"product_name": "Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:8",
			"fix_state": "Affected",
			"package_name": "bluez",
			"product_name": "Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:9",
			"fix_state": "Not affected",
			"package_name": "bluez",
			"product_name": "Red Hat Enterprise Linux 9"
		}
	],
	"upstream_fix": "bluez 5.56",
	"csaw": false
}
